models/book.js

```diff
@@ -0,0 +1,57 @@
+//vars used for global scope availability
+var app = app || {};
+// noinspection JSVoidFunctionReturnValueUsed
+app.BookModel = Backbone.Model.extend(
+	{
+		initialize : function (options) {
+		},
+		/**
+		 * Sets url path for API requests
+		 */
+		url: function () {
+			let myURL = document.location.origin + '/wp-json/bookAPI/v1/book';
+			if (this.id) {
+				myURL += '/' + this.id;
+			}
+			return myURL;
+		},
+		/**
+		 * Default values<br>
+		 * post_id is by default the id of the current book instance
+		 */
+		defaults: {
+			post_id : jQuery( '#get_page_id' ).val(),
```

Comment:
pls give the `post_id` at initialise, when a new model is created but from what you localize from php.